diagnostic.md

```diff
@@ -7,40 +7,54 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git clone git@github.com:joebelmonte/git-diagnostic.git
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
 
-<!-- Remove this comment and place your answer here. -->
+git init
```

Comment:
Thanks - I was definitely confused about this earlier today, but I realized what was going on over the course of the trainings today.  Now I understand.